security/pkg/nodeagent/tools/sds_cli.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
Suggest to move this to security/cmd/sds_cli/main.go?

Reply:
I think it's ok to put under tools since it's just for testing now(btw see pilot's example :)

Reply:
Pilot puts it under a separate folder, not `pilot/pkg` path.

https://github.com/istio/istio/tree/master/pilot/tools/debug

Reply:
moved it to security/tools/sds